packages/plugins/layout/parallax-background/src/index.js

```diff
@@ -62,14 +57,14 @@ class PluginComponent extends Component {
               label="Image location (URL)"
               style={{ width: '256px' }}
               value={background}
-              onChange={handleChange(onChange, 'background')}
-            />
+              onChange={e => onChange({ background: e.target.value })}
```

Comment:
Defining functions in render methods is bad practice as it creates a new function on every render. Instead, please modify the `handleChange` function!

Reply:
Thanks that's actually right ... hmm I think I will have to look into this. Do you know how much this hinders performance? Cause I use this all the time

Reply:
In case of this editor it's significant

Reply:
The function should probably be:

```
const handleChange = (onChange: (state: any) => void, key: string) => (
   e: Event,
 ) => onChange({ [key]: e.target.Value })
```

Although `Event` should probably be SyntheticEvent or SyntheticInputEvent (not sure which one of the two without flow to check atm)